app/backend/src/ui/transport-server/handlers/workspace/get-workspace.ts

```diff
@@ -0,0 +1,21 @@
+import { MessageHandler } from '../messageHandler';
+import { getWorkspace } from 'application/workspace/get-workspace';
+import { GetWorkspaceResponsePayload } from 'types/transport/responses/workspace/get-workspace';
+import { GetWorkspaceMessagePayload } from 'types/transport/requests/workspace/get-workspace';
+
+/**
+ * Get workspace handler
+ */
+export class GetWorkspaceHandler implements MessageHandler {
+  public type = 'get-workspace';
+
+  /**
+   * @param payload - payload from message to parse
+   */
+  // eslint-disable-next-line no-unused-vars
+  public async handle(payload: GetWorkspaceMessagePayload | ''): Promise<GetWorkspaceResponsePayload> {
```

Comment:
the payload is not used?

Reply:
I actually have no idea what to do since there is only one workspace the payload is redundant but should use because the payload is required parameter of handle function. Should i update `MessageHandler`'s handle function to allow empty parameter?

Reply:
yes, I think it should be optional

Reply:
Made payload as optional.